src/OrchardCore/OrchardCore/Localization/NullLocalizerFactory.cs

```diff
@@ -17,7 +23,7 @@ internal class NullLocalizerFactory : IStringLocalizerFactory, IHtmlLocalizerFac
 
         IHtmlLocalizer IHtmlLocalizerFactory.Create(Type resourceSource) => NullLocalizer.Instance;
 
-        private class NullLocalizer : IStringLocalizer, IHtmlLocalizer
+        public class NullLocalizer : IStringLocalizer, IHtmlLocalizer
```

Comment:
I dislike this change, if the intent to use it in the test, you can use The `IHtmlLocalizer` instance from the DI instead